asreviewcontrib/insights/plot.py

```diff
@@ -69,7 +69,7 @@ def _plot_recall(ax, labels, x_absolute=False, y_absolute=False):
         yticks = [0, 0.2, 0.4, 0.6, 0.8, 1.0]
 
     ax.step(x, y, where='post')
-    ax.set_title("Recall")
+    ax.set_title("Recall plot")
```

Comment:
Remove plot? Or use plot with the others as well? I prefer the first.

Reply:
Recall is already used on the y-axis. So, what is the title of this graph?

Reply:
The same holds for ERF. We can add the project name to it later.

Reply:
Alternative: Recall/erf at every step of the active learning process

Reply:
see new suggestion below